PublisherView.js

```diff
@@ -81,6 +81,12 @@ class PublisherView extends React.Component {
   sayHello() {
     OpenTokPublisherViewManager.sayHello();
   }
```

Comment:
Yeah, feel free to remove it here and we can remove it from the POC in another PR there.